### ISSUE TITLE
winget 1.3.431-preview - upgrade --include-unknown does not include anything more

### ISSUE THREAD

**Issue by o-l-a-v** [microsoft/winget-cli#2022]
### Brief description of your issue

Trying out winget [1.3.431-preview](https://github.com/microsoft/winget-cli/releases/tag/v1.3.431).

```winget upgrade```

* Notice the last line.

```
Name            Id              Version      Available   Source
---------------------------------------------------------------
Git             Git.Git         2.20.1       2.35.1.2    winget
HexChat         HexChat.HexChat 2.14.2       2.16.1      winget
Microsoft Teams Microsoft.Teams 1.4.00.23957 1.5.00.4689 winget
Authy Desktop   Twilio.Authy    1.9.0        2.0.0       winget
Microsoft Teams Microsoft.Teams 1.2.0.24753  1.5.00.4689 winget
5 upgrades available.
1 package has a version number that cannot be determined. Use "--include-unknown" to see all results.
```

<details><summary>Logs</summary>

```
2022-03-13 20:42:44.603 [CORE] WinGet, version [1.3.0-preview], activity [{0200461E-436F-49F4-88CA-E0E2A6BB0CB0}]
2022-03-13 20:42:44.606 [CORE] OS: Windows.Desktop v10.0.19044.1586
2022-03-13 20:42:44.606 [CORE] Command line Args: winget  upgrade
2022-03-13 20:42:44.606 [CORE] IsCOMCall:0; Caller: winget-cli
2022-03-13 20:42:44.607 [CLI ] WinGet invoked with arguments: 'upgrade'
2022-03-13 20:42:44.607 [CLI ] Found subcommand: upgrade
2022-03-13 20:42:44.607 [CLI ] Leaf command to execute: root:upgrade
2022-03-13 20:42:44.608 [CLI ] Executing command: upgrade
2022-03-13 20:42:44.611 [REPO] GetCurrentSourceRefs: Source named 'microsoft.builtin.desktop.frameworks' from origin Default is hidden and is dropped.
2022-03-13 20:42:44.611 [REPO] Default source requested, multiple sources available, adding all to source references.
2022-03-13 20:42:44.611 [REPO] Adding to source references msstore
2022-03-13 20:42:44.611 [REPO] Adding to source references winget
2022-03-13 20:42:44.614 [REPO] Multiple sources available, creating aggregated source.
2022-03-13 20:42:44.614 [REPO] Adding to aggregated source: msstore
2022-03-13 20:42:44.614 [REPO] Sending http GET request to: https://storeedgefd.dsx.mp.microsoft.com/v9.0/information
2022-03-13 20:42:44.832 [REPO] Response status: 200
2022-03-13 20:42:44.832 [REPO] Sending http GET request to: https://storeedgefd.dsx.mp.microsoft.com/v9.0/information
2022-03-13 20:42:44.926 [REPO] Response status: 200
2022-03-13 20:42:44.926 [REPO] Adding to aggregated source: winget
2022-03-13 20:42:44.927 [REPO] Opening SQLite Index for Read at 'C:\Users\olavb\AppData\Local\Microsoft/WinGet\State\Microsoft.PreIndexed.Package\Microsoft.Winget.Source_8wekyb3d8bbwe\index.db'
2022-03-13 20:42:44.927 [SQL ] Opening SQLite connection: 'C:\Users\olavb\AppData\Local\Microsoft/WinGet\State\Microsoft.PreIndexed.Package\Microsoft.Winget.Source_8wekyb3d8bbwe\index.db' [1, 0]
2022-03-13 20:42:44.929 [REPO] Opened SQLite Index with version [1.3], last write [2022-03-13 16:11:00.000]
2022-03-13 20:42:45.260 [REPO] Creating PredefinedInstalledSource with filter [None]
2022-03-13 20:42:45.260 [REPO] Creating new SQLite Index [4294967295.4294967295] at ':memory:'
2022-03-13 20:42:45.260 [SQL ] Opening SQLite connection: ':memory:' [6, 0]
2022-03-13 20:42:45.323 [REPO] Examining ARP entries for Machine | X64
2022-03-13 20:42:45.378 [REPO] Examining ARP entries for Machine | X86
2022-03-13 20:42:45.438 [REPO] Examining ARP entries for User | X64
2022-03-13 20:42:45.895 [REPO] Opening SQLite Index for ReadWrite at 'C:\Users\olavb\AppData\Local\Microsoft/WinGet\State\StoreEdgeFD\installed.db'
2022-03-13 20:42:45.895 [SQL ] Opening SQLite connection: 'C:\Users\olavb\AppData\Local\Microsoft/WinGet\State\StoreEdgeFD\installed.db' [2, 0]
2022-03-13 20:42:45.896 [REPO] Opened SQLite Index with version [1.4], last write [2022-03-13 20:34:16.000]
2022-03-13 20:42:45.956 [REPO] Opening SQLite Index for ReadWrite at 'C:\Users\olavb\AppData\Local\Microsoft/WinGet\State\Microsoft.Winget.Source_8wekyb3d8bbwe\installed.db'
2022-03-13 20:42:45.956 [SQL ] Opening SQLite connection: 'C:\Users\olavb\AppData\Local\Microsoft/WinGet\State\Microsoft.Winget.Source_8wekyb3d8bbwe\installed.db' [2, 0]
2022-03-13 20:42:45.958 [REPO] Opened SQLite Index with version [1.4], last write [2022-03-13 20:34:16.000]
2022-03-13 20:42:46.085 [REPO] Found multiple matches for installed package [Audacity_is1] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='audacity_is1'[Exact] Include:NormalizedNameAndPublisher='audacity'+'audacityteam'[Exact]]
2022-03-13 20:42:46.086 [REPO]   Checking match with package id: Audacity.Audacity
2022-03-13 20:42:46.088 [REPO]   Checking match with package id: Audacity.Audacity.Alpha
2022-03-13 20:42:46.287 [REPO] Found multiple matches for installed package [Mozilla Firefox 98.0 (x64 en-US)] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='mozilla firefox 98.0 (x64 en-us)'[Exact] Include:NormalizedNameAndPublisher='mozillafirefox'+'mozilla'[Exact]]
2022-03-13 20:42:46.289 [REPO]   Checking match with package id: Mozilla.Firefox.Beta
2022-03-13 20:42:46.291 [REPO]   Checking match with package id: Mozilla.Firefox
2022-03-13 20:42:46.291 [REPO]   Appropriate available package could not be determined
2022-03-13 20:42:46.440 [REPO] Found multiple matches for installed package [{1D8E6291-B0D5-35EC-8441-6616F567A0F7}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{1d8e6291-b0d5-35ec-8441-6616f567a0f7}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualc2010redistributable'+'microsoft'[Exact]]
2022-03-13 20:42:46.442 [REPO]   Checking match with package id: Microsoft.VC++2010Redist-x64
2022-03-13 20:42:46.443 [REPO]   Checking match with package id: Microsoft.VC++2010Redist-x86
2022-03-13 20:42:46.468 [REPO] Found multiple matches for installed package [{2aaf1df0-eb13-4099-9992-962bb4e596d1}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{2aaf1df0-eb13-4099-9992-962bb4e596d1}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualcredistributable'+'microsoft'[Exact]]
2022-03-13 20:42:46.470 [REPO]   Checking match with package id: Microsoft.VC++2015-2022Redist-x64
2022-03-13 20:42:46.471 [REPO]   Checking match with package id: Microsoft.VC++2015-2022Redist-x86
2022-03-13 20:42:46.473 [REPO]   Checking match with package id: Microsoft.VC++2015-2019Redist-x86
2022-03-13 20:42:46.475 [REPO]   Checking match with package id: Microsoft.VC++2015-2019Redist-x64
2022-03-13 20:42:46.499 [REPO] Found multiple matches for installed package [{41d7b770-418a-43b7-95a5-f925fff05789}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{41d7b770-418a-43b7-95a5-f925fff05789}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualcredistributable'+'microsoft'[Exact]]
2022-03-13 20:42:46.501 [REPO]   Checking match with package id: Microsoft.VC++2015-2022Redist-x86
2022-03-13 20:42:46.502 [REPO]   Checking match with package id: Microsoft.VC++2015-2022Redist-x64
2022-03-13 20:42:46.504 [REPO]   Checking match with package id: Microsoft.VC++2015-2019Redist-x86
2022-03-13 20:42:46.506 [REPO]   Checking match with package id: Microsoft.VC++2015-2019Redist-x64
2022-03-13 20:42:46.543 [REPO] Found multiple matches for installed package [{5FCE6D76-F5DC-37AB-B2B8-22AB8CEDB1D4}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{5fce6d76-f5dc-37ab-b2b8-22ab8cedb1d4}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualc2008redistributable'+'microsoft'[Exact]]
2022-03-13 20:42:46.545 [REPO]   Checking match with package id: Microsoft.VC++2008Redist-x64
2022-03-13 20:42:46.546 [REPO]   Checking match with package id: Microsoft.VC++2008Redist-x86
2022-03-13 20:42:46.570 [REPO] Found multiple matches for installed package [{710f4c1c-cc18-4c49-8cbf-51240c89a1a2}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{710f4c1c-cc18-4c49-8cbf-51240c89a1a2}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualc2005redistributable'+'microsoft'[Exact]]
2022-03-13 20:42:46.571 [REPO]   Checking match with package id: Microsoft.VC++2005Redist-x86
2022-03-13 20:42:46.572 [REPO]   Checking match with package id: Microsoft.VC++2005Redist-x64
2022-03-13 20:42:46.620 [REPO] Found multiple matches for installed package [{9BE518E6-ECC6-35A9-88E4-87755C07200F}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{9be518e6-ecc6-35a9-88e4-87755c07200f}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualc2008redistributable'+'microsoft'[Exact]]
2022-03-13 20:42:46.622 [REPO]   Checking match with package id: Microsoft.VC++2008Redist-x86
2022-03-13 20:42:46.623 [REPO]   Checking match with package id: Microsoft.VC++2008Redist-x64
2022-03-13 20:42:46.711 [REPO] Found multiple matches for installed package [{F0C3E5D1-1ADE-321E-8167-68EF0DE699A5}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{f0c3e5d1-1ade-321e-8167-68ef0de699a5}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualc2010redistributable'+'microsoft'[Exact]]
2022-03-13 20:42:46.713 [REPO]   Checking match with package id: Microsoft.VC++2010Redist-x86
2022-03-13 20:42:46.714 [REPO]   Checking match with package id: Microsoft.VC++2010Redist-x64
2022-03-13 20:42:46.728 [REPO] Found multiple matches for installed package [{ad8a2fa1-06e7-4b0d-927d-6e54b3d31028}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{ad8a2fa1-06e7-4b0d-927d-6e54b3d31028}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualc2005redistributable'+'microsoft'[Exact]]
2022-03-13 20:42:46.729 [REPO]   Checking match with package id: Microsoft.VC++2005Redist-x64
2022-03-13 20:42:46.730 [REPO]   Checking match with package id: Microsoft.VC++2005Redist-x86
2022-03-13 20:42:46.936 [CLI ] Leaf command succeeded: root:upgrade
```

</details>

```winget upgrade --include-unknown```

* Notice it still returns 5 packages only.

```
Name            Id              Version      Available   Source
---------------------------------------------------------------
Git             Git.Git         2.20.1       2.35.1.2    winget
HexChat         HexChat.HexChat 2.14.2       2.16.1      winget
Microsoft Teams Microsoft.Teams 1.4.00.23957 1.5.00.4689 winget
Authy Desktop   Twilio.Authy    1.9.0        2.0.0       winget
Microsoft Teams Microsoft.Teams 1.2.0.24753  1.5.00.4689 winget
5 upgrades available.
```

<details><summary>Logs</summary>

```
2022-03-13 20:43:54.989 [CORE] WinGet, version [1.3.0-preview], activity [{B28E63EB-F2A8-4F80-B056-E5B3B4CC38F7}]
2022-03-13 20:43:54.991 [CORE] OS: Windows.Desktop v10.0.19044.1586
2022-03-13 20:43:54.991 [CORE] Command line Args: winget  upgrade --include-unknown
2022-03-13 20:43:54.991 [CORE] IsCOMCall:0; Caller: winget-cli
2022-03-13 20:43:54.992 [CLI ] WinGet invoked with arguments: 'upgrade' '--include-unknown'
2022-03-13 20:43:54.992 [CLI ] Found subcommand: upgrade
2022-03-13 20:43:54.993 [CLI ] Leaf command to execute: root:upgrade
2022-03-13 20:43:54.994 [CLI ] Executing command: upgrade
2022-03-13 20:43:54.997 [REPO] GetCurrentSourceRefs: Source named 'microsoft.builtin.desktop.frameworks' from origin Default is hidden and is dropped.
2022-03-13 20:43:54.997 [REPO] Default source requested, multiple sources available, adding all to source references.
2022-03-13 20:43:54.997 [REPO] Adding to source references msstore
2022-03-13 20:43:54.997 [REPO] Adding to source references winget
2022-03-13 20:43:55.000 [REPO] Multiple sources available, creating aggregated source.
2022-03-13 20:43:55.000 [REPO] Adding to aggregated source: msstore
2022-03-13 20:43:55.000 [REPO] Sending http GET request to: https://storeedgefd.dsx.mp.microsoft.com/v9.0/information
2022-03-13 20:43:55.223 [REPO] Response status: 200
2022-03-13 20:43:55.224 [REPO] Sending http GET request to: https://storeedgefd.dsx.mp.microsoft.com/v9.0/information
2022-03-13 20:43:55.330 [REPO] Response status: 200
2022-03-13 20:43:55.330 [REPO] Adding to aggregated source: winget
2022-03-13 20:43:55.331 [REPO] Opening SQLite Index for Read at 'C:\Users\olavb\AppData\Local\Microsoft/WinGet\State\Microsoft.PreIndexed.Package\Microsoft.Winget.Source_8wekyb3d8bbwe\index.db'
2022-03-13 20:43:55.331 [SQL ] Opening SQLite connection: 'C:\Users\olavb\AppData\Local\Microsoft/WinGet\State\Microsoft.PreIndexed.Package\Microsoft.Winget.Source_8wekyb3d8bbwe\index.db' [1, 0]
2022-03-13 20:43:55.333 [REPO] Opened SQLite Index with version [1.3], last write [2022-03-13 16:11:00.000]
2022-03-13 20:43:55.634 [REPO] Creating PredefinedInstalledSource with filter [None]
2022-03-13 20:43:55.634 [REPO] Creating new SQLite Index [4294967295.4294967295] at ':memory:'
2022-03-13 20:43:55.634 [SQL ] Opening SQLite connection: ':memory:' [6, 0]
2022-03-13 20:43:55.697 [REPO] Examining ARP entries for Machine | X64
2022-03-13 20:43:55.750 [REPO] Examining ARP entries for Machine | X86
2022-03-13 20:43:55.807 [REPO] Examining ARP entries for User | X64
2022-03-13 20:43:56.262 [REPO] Opening SQLite Index for ReadWrite at 'C:\Users\olavb\AppData\Local\Microsoft/WinGet\State\StoreEdgeFD\installed.db'
2022-03-13 20:43:56.262 [SQL ] Opening SQLite connection: 'C:\Users\olavb\AppData\Local\Microsoft/WinGet\State\StoreEdgeFD\installed.db' [2, 0]
2022-03-13 20:43:56.263 [REPO] Opened SQLite Index with version [1.4], last write [2022-03-13 20:34:16.000]
2022-03-13 20:43:56.323 [REPO] Opening SQLite Index for ReadWrite at 'C:\Users\olavb\AppData\Local\Microsoft/WinGet\State\Microsoft.Winget.Source_8wekyb3d8bbwe\installed.db'
2022-03-13 20:43:56.323 [SQL ] Opening SQLite connection: 'C:\Users\olavb\AppData\Local\Microsoft/WinGet\State\Microsoft.Winget.Source_8wekyb3d8bbwe\installed.db' [2, 0]
2022-03-13 20:43:56.325 [REPO] Opened SQLite Index with version [1.4], last write [2022-03-13 20:34:16.000]
2022-03-13 20:43:56.452 [REPO] Found multiple matches for installed package [Audacity_is1] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='audacity_is1'[Exact] Include:NormalizedNameAndPublisher='audacity'+'audacityteam'[Exact]]
2022-03-13 20:43:56.453 [REPO]   Checking match with package id: Audacity.Audacity
2022-03-13 20:43:56.454 [REPO]   Checking match with package id: Audacity.Audacity.Alpha
2022-03-13 20:43:56.652 [REPO] Found multiple matches for installed package [Mozilla Firefox 98.0 (x64 en-US)] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='mozilla firefox 98.0 (x64 en-us)'[Exact] Include:NormalizedNameAndPublisher='mozillafirefox'+'mozilla'[Exact]]
2022-03-13 20:43:56.654 [REPO]   Checking match with package id: Mozilla.Firefox.Beta
2022-03-13 20:43:56.656 [REPO]   Checking match with package id: Mozilla.Firefox
2022-03-13 20:43:56.656 [REPO]   Appropriate available package could not be determined
2022-03-13 20:43:56.804 [REPO] Found multiple matches for installed package [{1D8E6291-B0D5-35EC-8441-6616F567A0F7}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{1d8e6291-b0d5-35ec-8441-6616f567a0f7}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualc2010redistributable'+'microsoft'[Exact]]
2022-03-13 20:43:56.806 [REPO]   Checking match with package id: Microsoft.VC++2010Redist-x64
2022-03-13 20:43:56.807 [REPO]   Checking match with package id: Microsoft.VC++2010Redist-x86
2022-03-13 20:43:56.831 [REPO] Found multiple matches for installed package [{2aaf1df0-eb13-4099-9992-962bb4e596d1}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{2aaf1df0-eb13-4099-9992-962bb4e596d1}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualcredistributable'+'microsoft'[Exact]]
2022-03-13 20:43:56.833 [REPO]   Checking match with package id: Microsoft.VC++2015-2022Redist-x64
2022-03-13 20:43:56.835 [REPO]   Checking match with package id: Microsoft.VC++2015-2022Redist-x86
2022-03-13 20:43:56.836 [REPO]   Checking match with package id: Microsoft.VC++2015-2019Redist-x86
2022-03-13 20:43:56.838 [REPO]   Checking match with package id: Microsoft.VC++2015-2019Redist-x64
2022-03-13 20:43:56.862 [REPO] Found multiple matches for installed package [{41d7b770-418a-43b7-95a5-f925fff05789}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{41d7b770-418a-43b7-95a5-f925fff05789}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualcredistributable'+'microsoft'[Exact]]
2022-03-13 20:43:56.864 [REPO]   Checking match with package id: Microsoft.VC++2015-2022Redist-x86
2022-03-13 20:43:56.866 [REPO]   Checking match with package id: Microsoft.VC++2015-2022Redist-x64
2022-03-13 20:43:56.868 [REPO]   Checking match with package id: Microsoft.VC++2015-2019Redist-x86
2022-03-13 20:43:56.869 [REPO]   Checking match with package id: Microsoft.VC++2015-2019Redist-x64
2022-03-13 20:43:56.906 [REPO] Found multiple matches for installed package [{5FCE6D76-F5DC-37AB-B2B8-22AB8CEDB1D4}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{5fce6d76-f5dc-37ab-b2b8-22ab8cedb1d4}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualc2008redistributable'+'microsoft'[Exact]]
2022-03-13 20:43:56.908 [REPO]   Checking match with package id: Microsoft.VC++2008Redist-x64
2022-03-13 20:43:56.909 [REPO]   Checking match with package id: Microsoft.VC++2008Redist-x86
2022-03-13 20:43:56.933 [REPO] Found multiple matches for installed package [{710f4c1c-cc18-4c49-8cbf-51240c89a1a2}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{710f4c1c-cc18-4c49-8cbf-51240c89a1a2}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualc2005redistributable'+'microsoft'[Exact]]
2022-03-13 20:43:56.934 [REPO]   Checking match with package id: Microsoft.VC++2005Redist-x86
2022-03-13 20:43:56.935 [REPO]   Checking match with package id: Microsoft.VC++2005Redist-x64
2022-03-13 20:43:56.983 [REPO] Found multiple matches for installed package [{9BE518E6-ECC6-35A9-88E4-87755C07200F}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{9be518e6-ecc6-35a9-88e4-87755c07200f}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualc2008redistributable'+'microsoft'[Exact]]
2022-03-13 20:43:56.985 [REPO]   Checking match with package id: Microsoft.VC++2008Redist-x86
2022-03-13 20:43:56.986 [REPO]   Checking match with package id: Microsoft.VC++2008Redist-x64
2022-03-13 20:43:57.073 [REPO] Found multiple matches for installed package [{F0C3E5D1-1ADE-321E-8167-68EF0DE699A5}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{f0c3e5d1-1ade-321e-8167-68ef0de699a5}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualc2010redistributable'+'microsoft'[Exact]]
2022-03-13 20:43:57.075 [REPO]   Checking match with package id: Microsoft.VC++2010Redist-x86
2022-03-13 20:43:57.076 [REPO]   Checking match with package id: Microsoft.VC++2010Redist-x64
2022-03-13 20:43:57.090 [REPO] Found multiple matches for installed package [{ad8a2fa1-06e7-4b0d-927d-6e54b3d31028}] in source [Microsoft.Winget.Source_8wekyb3d8bbwe] when searching for [Query:[none] Include:ProductCode='{ad8a2fa1-06e7-4b0d-927d-6e54b3d31028}'[Exact] Include:NormalizedNameAndPublisher='microsoftvisualc2005redistributable'+'microsoft'[Exact]]
2022-03-13 20:43:57.092 [REPO]   Checking match with package id: Microsoft.VC++2005Redist-x64
2022-03-13 20:43:57.093 [REPO]   Checking match with package id: Microsoft.VC++2005Redist-x86
2022-03-13 20:43:57.294 [CLI ] Leaf command succeeded: root:upgrade
```

</details>

### Steps to reproduce

See above.

### Expected behavior

One more package with unknown version number should be shown in the output.

### Actual behavior

The package with unknown version number is not shown in the output

### Environment

```
Windows Package Manager (Preview) v1.3.0-preview
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.19044.1586

Logs: %TEMP%\WinGet
```

**Comment by jedieaston** [comment]
This was resolved by https://github.com/microsoft/winget-cli/pull/1972 I believe, once a new preview release is cut you should be able to test this change.

**Comment by kiddliu** [comment]
Can still be reproduced with v1.3.692-preview

```
Windows Package Manager (Preview) v1.3.692-preview
Copyright (c) Microsoft Corporation. All rights reserved.

Windows: Windows.Desktop v10.0.22610.1
Package: Microsoft.DesktopAppInstaller v1.18.692.0

Logs: %LOCALAPPDATA%\Packages\Microsoft.DesktopAppInstaller_8wekyb3d8bbwe\LocalState\DiagOutputDir

Links
---------------------------------------------------------------------------
Privacy Statement   https://aka.ms/winget-privacy
License Agreement   https://aka.ms/winget-license
Third Party Notices https://aka.ms/winget-3rdPartyNotice
Homepage            https://aka.ms/winget
Windows Store Terms https://www.microsoft.com/en-us/storedocs/terms-of-sale
```

**Comment by Trenly** [comment]
@denelon - I believe this was resolved when [1.3.1872](https://github.com/microsoft/winget-cli/releases/tag/v1.3.1872) was released
